app/views/admin/editions/_html_version_fields.html.erb

```diff
@@ -0,0 +1,12 @@
+<h3 class=govuk-heading-l>HTML attachment</h3>
```

Comment:
This HTML attribute is missing double quotes around the value:

```suggestion
<h3 class="govuk-heading-l">HTML attachment</h3>
```

While it's [not technically necessary](https://stackoverflow.com/questions/6495310/do-you-quote-html5-attributes) in this case, it's a good habit to use them everywhere.